packages/gatsby/package.json

```diff
@@ -30,6 +30,7 @@
     "babel-plugin-dynamic-import-node": "^1.2.0",
     "babel-plugin-macros": "^2.4.0",
     "babel-plugin-remove-graphql-queries": "^2.0.2-rc.3",
+    "better-opn": "^0.1.0",
```

Comment:
Is there value in keeping `opn` or should we just always use `better-opn`?

Looks like it's used in the serve task as well.

Reply:
@DSchau We should always use `better-opn`. However, I make `opn` as a peer dependency at `better-opn` (check [here[(https://github.com/ExiaSR/better-opn/blob/master/package.json#L32), `opn` still have to be explicitly added as a dependency.